### ISSUE TITLE
new version

### ISSUE THREAD

**Issue by chorny** [happy-barney/perl-SQL-Admin#5]
CPAN still has old version with incorrect dependency name.

**Comment by jkeenan** [comment]
> CPAN still has old version with incorrect dependency name.

Yes, I was burnt by this today ... 7 years after this bug was first reported. @happy-barney, please fix!
